README.md

```diff
@@ -65,6 +64,48 @@ Values include: { A-Record | C-Name }
 
 * Added Resource xDnsRecord with support for CNames. 
 This will replace xDnsARecord in a future release.
+* Added Resource xDnsServerForwarder.
+
+#### Resources
+
+* **xDnsServerForwarder** sets a DNS forwarder on a given DNS server.
+
+##### xDnsServerForwarder
+
+* **IsSingleInstance**: Specifies the resource is a single instance, the value must be 'Yes'
+* **IPAddresses**: IP addresses of the forwarders
+
+### Examples
+
+#### Setting DNS Forwarders
+
+```powershell
+configuration Sample_Set_Forwarders
+{
+    Import-DscResource -module xDnsServer
+    xDnsServerForwarder SetForwarders
+    {
+    	IsSingleInstance = 'Yes'
+        IPAddresses = '192.168.0.10','192.168.0.11'
+    }
+}
+Sample_Set_Forwarders
+```
+
+#### Removing All DNS Forwarders
+
+```powershell
+configuration Sample_Remove_All_Forwarders
+{
+    Import-DscResource -module xDnsServer
+    xDnsServerForwarder RemoveAllForwarders
+    {
+        IsSingleInstance = 'Yes'
+        IPAddresses = @()
+    }
+}
+Sample_Remove_All_Forwarders
+```
```

Comment:
Please include a summary of changes in an unreleased version change per the [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#lifecycle-of-a-pull-request)